### DISCUSSION TITLE
Train from scratch ( default label colors)

### DISCUSSION THREAD

**Discussion by RotoDrache** [Project-MONAI/MONAILabel#1283] [Q&A] [answered]
I started a train from scratch session. I'm using the segmentation model. when I start the training it assigns label colors. Is there a way to set the colors for a model so that if I share the model the label colors will be the same. i.e. I have a label called 'ivc' and I want it a certain shade of blue when I train but also when I share this model with others.
can you please advise.

**Comment by tangy5** [comment, marked as answer]
As long as the number and order of classes of your model are the same, 3D Slicer can load and assign same color, or adjust. 

For example, the labels are defined here in this Radiology app, and classes are fixed: 

https://github.com/Project-MONAI/MONAILabel/blob/8ae10131c381d3436899086f8f834ceb937eb077/sample-apps/radiology/lib/configs/segmentation.py#L37

**Reply by RotoDrache** [reply]
Does the "1" after spleen in your example a color assignment. If not where is the color assignment your example segmentation.py?

**Reply by tangy5** [reply]
Color assginments are decided by Slicer. You could refer to the Slicer side. 
For example, the MONAI Label plugin has some general anatomy color here:

https://github.com/Project-MONAI/MONAILabel/blob/main/plugins/slicer/MONAILabel/MONAILabelLib/labelcolors.py

**Comment by RotoDrache** [comment]
Super thank,  thats what I was looking for I made the mistake of coming up with my own label names. I'll rerun with "standard" names in the labelcolors.py
once again many thanks
